### PR TITLE
Do not fail when handling Twitter 503 responses

### PR DESCRIPTION
**Pull request by orottier**
The body returned may be empty

### CODE REVIEW

**Comment by thujohn** [comment]
Added in 2.1.5

File: src/Thujohn/Twitter/Twitter.php [hunk 1]
```diff
@@ -271,15 +271,23 @@ public function query($name, $requestMethod = 'GET', $parameters = [], $multipar
...
-				$error_msg = $_response['errors'][0]['message'];
+				if (array_key_exists('errors', $_response))
+				{
+					$error_code = $_response['errors'][0]['code'];
+					$error_msg = $_response['errors'][0]['message'];
+				}
+				else
+				{
+					$error_code = $response['code'];
+					$error_msg = $_response['error'];
```
Comment (YOzaz) [inline src/Thujohn/Twitter/Twitter.php]: Should be $error_msg = $response['error];
(note underscore dash)

File: composer.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
-    "name": "thujohn/twitter",
+    "name": "orottier/twitter",
```
Comment (YOzaz) [inline composer.json]: Is this correct?

**Comment by orottier** [comment]
It's not merged, only closed @YOzaz

**Pull request closed without merging**